controllers/akodeploymentconfig/user/user_controller.go

```diff
@@ -17,6 +17,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/utils/pointer"
```

Comment:
nit: looks like a couple more `pointer.BoolPtr`s left that could be removed

Reply:
Thanks. I moved out all the test change to this repo https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/pull/241